.github/workflows/build.yml

```diff
@@ -17,19 +17,13 @@ jobs:
             fail-fast: false
             matrix:
                 include:
-                    -
-                        php: 7.4
-                        symfony-version: 4.4.*
-                    -
-                        php: 7.4
-                        symfony-version: 5.1.*
                     -
                         php: 8.0
                         symfony-version: 4.4.*
                         composer-flags: "--ignore-platform-req=php"
```

Comment:
can we drop this flag?

Reply:
Should we drop only this flag or in both symfony-version: 4.4.* and symfony-version: 5.4.* configs?